docs/examples/payment_card_number.py

```diff
@@ -0,0 +1,33 @@
+import calendar
+from datetime import date
+
+from pydantic import BaseModel
+from pydantic.types import PaymentCardBrand, PaymentCardNumber
+
+class Card(BaseModel):
+    name: str
+    number: PaymentCardNumber
+    exp: date
+
+    @property
+    def brand(self) -> PaymentCardBrand:
+        return self.number.brand
+
+    @property
+    def expired(self) -> bool:
+        return self.exp < date.today()
+
+def last_day_of_month(year: int, month: int) -> date:
```

Comment:
I don't think we need this method in the docs example - it's more complication than we need.